queries/cpp/highlights.scm

```diff
@@ -101,3 +101,5 @@
 ] @keyword
 
 "::" @operator
+
+((identifier) @type (#match? @type "^[A-Z]"))
```

Comment:
to the top of the file? this query is only a first guess and in some situations we know better.